csfieldguide/chapters/content/en/coding-error-control/sections/whats-the-big-picture.md

```diff
@@ -1,24 +1,25 @@
 # What's the big picture?
 
 There are several ways that data can be changed accidentally.
-Networks that have a lot of "noise" on them (caused by poor quality wiring, electrical interference, or interference from other networks in the case of wireless).
-The bits on disks are very very small, and imperfections in the surface can eventually cause some of the storage to fail.
-The surfaces on compact disks and DVDs are exposed, and can easily be damaged by storage (e.g. in heat or humidity) and handling (e.g. scratches or dust).
+Networks that have a lot of "noise" in them (caused by poor quality wiring, electrical interference, or interference from other networks).
+The {glossary-link term="bit"}bits{glossary-link end} on disks are very very small, and imperfections in the surface can eventually cause some of the storage to fail.
+The surfaces of CDs and DVDs are exposed, and can easily be damaged by storage (e.g. in heat or humidity) and handling (e.g. scratches or dust).
 Errors can also occur when numbers are typed in, such as entering a bank account number to make a payment into, or the number of a container that is being loaded onto a ship.
 A barcode on a product might be slightly scratched or have a black mark on it, or perhaps the package is bent or is unable to be read properly due to the scanner being waved too fast over it.
 Bits getting changed on permanent storage (such as hard drives, optical disks, and solid state drives) is sometimes referred to as data rot, and the [wikipedia page on bit rot](https://en.wikipedia.org/wiki/Data_degradation) has a list of more ways that these errors can occur.
 
-Nobody wants a computer that is unreliable and won’t do what it's supposed to do because of bits being changed!
+Nobody wants a computer that is unreliable and won’t do what it's supposed to do in a real-world scenario!
```

Comment:
Unsure about this change. Reason?

Reply:
I didn't like "bits being changed" because it's too abstract - the focus is on the opinion of nobody(everybody) and very few people are going to think bits being changed is the reason for their frustration. However I think I got confused and replaced what I replaced it with with the real-world scenario bit so I'll think more about it